adds new collection "network" and new role

privateip opened the pull request:
This commit adds a new collection `pureport.network` and adds the first
role to that collection `pureport.network.google_cloud_interconnect`.

Commit:
adds new collection "network" and new role

This commit adds a new collection `pureport.network` and adds the first
role to that collection `pureport.network.google_cloud_interconnect`.

mtraynham commented on network/roles/google_cloud_interconnect/tasks/main.yaml:
```diff
@@ -0,0 +1,2 @@
+---
+- include_tasks: { file: "{{ state }}.yaml" }
```
Clever 😄

mtraynham commented on network/roles/google_cloud_interconnect/defaults/main.yaml:
```diff
@@ -0,0 +1,7 @@
+---
+gcp_network_name: vpc
```
Should `state: present` be defaulted here?

mtraynham reviewed (changes requested):
Just 1 review comment.  I know the build failed, but I believe that's unrelated and I'll get a fix together for it.

Commit:
fixes issuew with galaxy.yml

This commit fixes the bug with galaxy.yml needing a dict not a list.  It
also sets the default value for `state` to `present`
